### ISSUE TITLE
France - CTDs on loading saved games 

### ISSUE THREAD

**Issue by Laiders72** [Darkest-Hour-Hoi4/Darkest-Hour-Issues#132]
### OS

Windows 11

### Any Expansions?

Together for Victory, Death or Dishonor, Waking the Tiger, Man the Guns, La Résistance, Battle For The Bosporus, No Step Back

### Are you using Steam?

Yes i am

### Were you playing in multiplayer?

No i wasn't

### Hoi4 Version

Avalanche v1.12.14.50e7

### DH Version (Current one is 1.0.4-hotfix1)

1.0.1-hotfix (Steam version)

### Other Mods

Songs of Our Darkest Hour

### Description

```bash
I have two possibly connected issues. The major issue is that the game consistently crashes on loading saves. These crashes occur not on loading but on zooming in to the point unit models are loaded. At this point, the game immediately crashes. In theory, the save would remain stable if I stayed above this critical point of zoom. This does make it quite difficult to visualise combat and is easy to forget.

As a related point, some unit models just staight up do not work for my configuration. The Italians in particular have a habit of attacking me with ghostly rifles. 

I seem to be able to play games of arbitary length. The bug or misinstall etc. only seems to be fatal on loading a save, specifically loading unit models after a save. At a certain point, some units do turn into ghostly rifles though.

I have attempted to remedy this issue by: 

1. Validating game files

2. Fully deleting the mod and forcing Steam to redownload it. This was done by deleting the appworkshop_394360 file and the related folder. 

3. Disabling unit packs in case there is a conflict.

I have yet to check if there are options for disabling unit models in the game and replacing them with static symbols. I am not sure if this is even an option in HOI4 as I have never had to look for it.
```

### Reproduction steps

```bash
1. Load the save attached
2. Zoom to the point unit models load
3. For me, the game always crashes at this point
```

### Logs

[error.log](https://github.com/Darkest-Hour-Hoi4/Darkest-Hour-Issues/files/11709160/error.log)
[system.log](https://github.com/Darkest-Hour-Hoi4/Darkest-Hour-Issues/files/11709161/system.log)
[code_revisions.log](https://github.com/Darkest-Hour-Hoi4/Darkest-Hour-Issues/files/11709162/code_revisions.log)

Attached are the most recent crash logs from 19:04 today. Please let me know if you need any other files from the crash folder. I did try to uphold all of them but Github would only let me attach individual files. I have 12 more crash dumps. 

### Savegame

[FRA_1939_07_02_01.zip](https://github.com/Darkest-Hour-Hoi4/Darkest-Hour-Issues/files/11709185/FRA_1939_07_02_01.zip)

Latest save

**Comment by Laiders72** [comment]
I can confirm turning all 3D models off has resolved the crashes. I assume it is something wrong with my specific install or config. I am happy to provide a list of unit packs if they might be causing a problem.

**Comment by Darijus11** [comment]
We've already identified the issue and fixed it. We will deploy the fix with 1.0.5 version. The 1.0.5 version will be deployed when ready. In the meantime, keep 3D units off. Thank you for reporting!
